lib/windup.js

```diff
@@ -52,7 +53,7 @@ Windup.prototype.move = function(floors) {
     this.x += this.paceRate;
   }
 
-  if (this.status !== "falling" && !new Jump().onAFloor(floors, this)) {
+  if (this.status !== "falling" && !this.jump.onAFloor(floors, this)) {
     this.y += 2;
```

Comment:
This seems a little bit odd when I read it. Ideally I would like the code to read something like dino.onAFloor